user/asciinema/template.py

```diff
@@ -0,0 +1,22 @@
+pkgname = "asciinema"
+pkgver = "2.4.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+depends = [
+    "python",
+    "ncurses",
```

Comment:
what's the purpose of depending on ncurses package, does it need some of the programs?

Reply:
it used to use tput and whatnot until 2.1, seems nobody read the changelog